plugins/genesis-tools/skills/research/references/mcps.md

```
@@ -0,0 +1,154 @@
+```
+
+CLI install: `claude mcp add --transport http jina https://mcp.jina.ai/v1 --header "Authorization: Bearer <YOUR_JINA_API_KEY>"`
+
+### brave-search
+
+Tools used by this skill: `mcp__brave-search__brave_web_search`, `mcp__brave-search__brave_local_search`.
+
+Local stdio server. User needs a Brave Search API key from <https://brave.com/search/api/>.
+
+Install:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)